Simulator/simulator/garden.py

```diff
@@ -1,10 +1,12 @@
 import numpy as np
 from heapq import nlargest
 from simulator.logger import Logger, Event
-#from simulator.visualization import setup_animation, setup_saving
-from simulator.sim_globals import MAX_WATER_LEVEL, IRRIGATION_AMOUNT, PERMANENT_WILTING_POINT, PRUNE_DELAY, PRUNE_THRESHOLD, NUM_IRR_ACTIONS, PRUNE_RATE
+# from simulator.visualization import setup_animation, setup_saving
+from simulator.sim_globals import MAX_WATER_LEVEL, IRRIGATION_AMOUNT, PERMANENT_WILTING_POINT, PRUNE_DELAY, \
+    PRUNE_THRESHOLD, NUM_IRR_ACTIONS, PRUNE_RATE
 import pickle
 import multiprocessing as mp
+import os
```

Comment:
Is this used?

Reply:
I don't believe so, I used this for growth rate analysis